test/DynamoCoreTests/DummyNodeTests.cs

```diff
@@ -88,5 +97,29 @@ public void WorkspaceIsReadonly_IfXmlDummyNodePresent()
             this.CurrentDynamoModel.CurrentWorkspace.RemoveAndDisposeNode(dummyNode);
             Assert.IsFalse(this.CurrentDynamoModel.CurrentWorkspace.IsReadOnly);
         }
+
+        [Test]
+        public void ResolveDummyNodesOnDownloadingPackage()
```

Comment:
please add a test that does this same workflow *inside* a custom node - IE - the current workspace should be a custom node with dummy nodes in it.

Reply:
Are there more complex tests you can think of that we might be missing? 
* dummy nodes connecting to dummy nodes?
* a test that asserts the view properties are transferred correctly? I don't see that in your test - ie check the name, position etc.

Reply:
Added the test for dummy nodes inside a custom node workspace